src/Components/TriggerEditForm/TriggerEditForm.jsx

```diff
@@ -46,6 +52,7 @@ export default class TriggerEditForm extends React.Component<Props, State> {
         const { targets, trigger_type: triggerType } = props.data;
         this.state = {
             advancedMode: targets.length > 1 || triggerType === "expression",
+            activeTab: "edit",
```

Comment:
`activeTab` is very abstract name. Let's change to something like `descriptionMode`